roles/overpass-api/templates/overpass-areas-builder.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Overpass areas builder
+Requires=overpass-dispatcher.service overpass-areas-dispatcher.service
+AssertPathExists={{ overpass_database_dir }}/replicate_id
+
+[Service]
+Type=simple
+User={{ user }}
+ExecStart=/usr/local/bin/rules_loop.sh {{ overpass_database_dir }}
+Nice=19
+Restart=always
+RestartSec=30s
```

Comment:
RestartSec is used when the service failled, in this case, when rules_loop.sh crash.
Actually, in this case it will never be hit as rules_loop.sh is mainly a `while true` loop.